<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>裁剪算法</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="MyLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRect"/>
   <addaction name="separator"/>
   <addaction name="actionPoly"/>
   <addaction name="separator"/>
   <addaction name="actionPolyD"/>
   <addaction name="separator"/>
   <addaction name="actionCurve"/>
   <addaction name="separator"/>
   <addaction name="actionNew"/>
  </widget>
  <action name="actionRect">
   <property name="text">
    <string>矩形</string>
   </property>
  </action>
  <action name="actionPoly">
   <property name="text">
    <string>取多边形</string>
   </property>
  </action>
  <action name="actionCurve">
   <property name="text">
    <string>裁剪</string>
   </property>
  </action>
  <action name="actionPolyD">
   <property name="text">
    <string>绘制多边形</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>重绘</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
